src/sources/soundsourceffmpeg.cpp

```diff
@@ -81,20 +99,38 @@ inline int64_t getStreamStartTime(const AVStream& avStream) {
     auto start_time = avStream.start_time;
     if (start_time == AV_NOPTS_VALUE) {
         // This case is not unlikely, e.g. happens when decoding WAV files.
-#if ENABLE_TRACING
-        kLogger.trace()
+        switch (avStream.codecpar->codec_id) {
+        case AV_CODEC_ID_AAC:
+        case AV_CODEC_ID_AAC_LATM: {
+            // Account for the expected decoder delay instead of simply
+            // using the default start time.
+            // Not all M4A files encode the start_time correctly, e.g.
+            // the test file cover-test-itunes-12.7.0-aac.m4a has a valid
+            // start_time of 0. Unfortunately, this special case is cannot
+            // detected and compensated.
+            start_time = math_max(kavStreamDefaultStartTime, kavStreamDecoderDelayAAC);
+            break;
+        }
+        default:
+            start_time = kavStreamDefaultStartTime;
```

Comment:
```suggestion
            startTime = kAvStreamDefaultStartTime;
```
Also, could you add the unit? Is it the time in frames/samples/milliseconds?

Reply:
Nope, this is just a filter for FFmpeg data. The time calculations of FFmpeg are complicated and the unit depends on the encoded format.

Reply:
Ah, ok.